components/text_sensor/ble_scanner.rst

```diff
@@ -0,0 +1,46 @@
+ESP32 Bluetooth Low Energy Scanner
+======================================
+
+.. seo::
+    :description: Instructions for setting up BLE text sensors for the ESP32.
+    :image: bluetooth.png
+    :keywords: ESP32
+
+The ``ble_scanner`` text sensor platform lets you track reachable BLE devices.
+See the `BLE Tracker Configuration variables <https://esphome.io/components/esp32_ble_tracker.html#configuration-variables>`__ for
```

Comment:
Use `ref`s here instead of the esphome.io link. See the contributing guide

In esp32_ble_tracker.rst, add an `.. _config-esp32_ble_tracker:` before the Configuration variables header.

Reply:
Thank you, I didn't want to cause conflicts by editing esp32_ble_tracker. Resolved